nucliadb/nucliadb/ingest/orm/shard.py

```diff
@@ -56,9 +57,19 @@ async def delete_resource(self, uuid: str, txid: int):
             indexpb.typemessage = IndexMessage.TypeMessage.DELETION
             await indexing.index(indexpb, shardreplica.node)
 
+    @staticmethod
+    def counters_are_different(
```

Comment:
For me it feels more as a method to have in `ShardCounter` instead of `Shard`

Reply:
Good point

Reply:
Be carefully though if a comparison is done between counters of shards with different ids (or they will always be different)

Reply:
IINW, the shard is is the logical shard id, as we're comparing counters from different replicas. I used the Shard.sharduuid for this, can you confirm is the logical one? (the one that does not change)

Reply:
yes, it's the logical one! looks good now